### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by nixj14**

### CODE REVIEW

File: etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java [hunk 1]
```diff
@@ -571,20 +597,12 @@ public int compareTo(VCFWalker o) {
 		}
 	}
 
+	private static final int FILE_COLUMN = 0, CHROMOSOME_COLUMN = 1, ANNOTATED_FLAG_COLUMN = 2, GZIP_FLAG_COLUMN = 3,
```
Comment (chunacatsunflower) [inline etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java:600]: I like these listed out one per line, so it's nice and easy to read at a glance.

**Review by chunacatsunflower — APPROVED** [review]
looks good to me.

File: etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java (hunk 1 above)
Comment (nixj14) [reply etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java:600]: yeah, I see the appeal.  Do you have a formatter or formatting config that you use?  We basically use the standard Eclipse conventions with minor tweaks (max line  width).  i'll see if there are any settings applicable here that might format things better.

Comment (nixj14) [reply etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java]: just updated to break each of those statements out.  couldn't figure out how to configure eclipse formatter to do what you were wanting.  this also helps to resolve some sonarlint findings as well (2 birds, 1 stone)

Comment (chunacatsunflower) [reply etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java]: eclipse will put them one per line if there is a unique type declaration, e.g.:

int FILE = 0;
int CHROM = 1;
int ANNOTATE = 2;  etc.

Comment (nixj14) [reply etl/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/etl/genotype/NewVCFLoader.java]: yep.  Do the changes look good now?

**Pull request merged**